hoomd/hpmc/validation/truncated_tetrahedra_alchemy.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+import hoomd
```

Comment:
This file does not belong in the HOOMD repository. Please keep it somewhere else for now. I will prepare a new `hoomd-validation` repository soon for tests like this.